src/containers/app/AppReducer.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * @flow
+ */
+
+import Immutable from 'immutable';
+
+import * as actionTypes from './AppActionTypes';
+
+export const INITIAL_STATE :Map<> = Immutable.fromJS({
+  apps: Immutable.List(),
+  errorMessage: '',
+  allOrganizations: Immutable.List(),
```

Comment:
`allOrganizations` and `ownedOrganizations` are not set anywhere in this reducer.

1. Do you need them?
2. I would think `allOrganizations` and `ownedOrganizations` belong in the Organizations reducer. Thoughts?

Reply:
oops I forgot to remove these, thanks!